packages/namekit-react/src/components/DomainOwnershipBadge.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import {
+  Address,
+  DomainCard,
+  getUserOwnership,
+  UserOwnershipOfDomain,
+} from "@namehash/ens-utils";
+import React from "react";
+import cc from "classcat";
+
+interface DomainOwnershipBadgeProps {
+  domain: DomainCard | null; // when null we show the skeleton
+  onClickHandler?: () => void;
+  addressToCompareOwnership: Address | null;
```

Comment:
Can you please suggest some more clear way to name and/or document this field? It's not clear for me how that field works just by reading the name.